cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java

```diff
@@ -70,7 +70,7 @@ private OkHttpClient createHttpClient() {
 
         try {
             Request request = new Request.Builder()
-                .url(cerberusUrl + "/v1/metadata?limit=2000&offset=0")
+                .url(cerberusUrl + "/v1/metadata?limit=10000&offset=0")
```

Comment:
What is this, this call should be a do while loop until there is no next page?

Reply:
```
def res = list
def hasNext = false
do {
 def cur = fetch the results
 res += cur.stuff you care about
 hasNext = cur.hasNext
} while there is no next page
```

Reply:
I think 10000 is fine until there's a problem. 
But then I just realized this is on public repo. 😅

Reply:
Will implement this